ext/spl/tests/gh9883.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Bug GH-9883 (SplFileObject::__toString() reads next line)
+--FILE--
+<?php
+$file_stream = new SplFileObject(__FILE__, 'rb');
+
+echo $file_stream; // line 4
```

Comment:
Why such comments? I would remove them.

Reply:
See #9883. The comments helped to identify what was going on, and may be helpful if something breaks (again) in the future.